content/livesync/index.textile

```diff
@@ -1,6 +1,57 @@
 ---
 title: What is LiveSync?
+product: livesync
+languages:
+  - javascript
 ---
 
+p. The LiveSync product consists of three parts: the "Ably Realtime Data Models SDK":https://https://github.com/ably-labs/models,  "Ably Database Connector (ADBC)":https://github.com/ably-labs/adbc and an "Outbox Table":https://docs.aws.amazon.com/prescriptive-guidance/latest/cloud-design-patterns/transactional-outbox.html. LiveSync's purpose is to allow you to easily broadcast and receive realtime updates of an app's state through data stored on a database. LiveSync can support multiple users with concurrent updates and offers an optimistic UI for publishers, and a 'last write wins' strategy for transactional updates.
```

Comment:
> through data stored on a database

Not sure this wording of "through" makes sense to me, how about perhaps: "LiveSync allows you to easily broadcast and receive realtime updates to your application state stored in your database"?

Reply:
Just a suggestion - I also quite like the wording in the [PowerSync](https://www.powersync.co/) landing page, which sells control over database writes as a value add:

> ... a default of last write wins. Developers can customize conflict resolution and are in full control of the write process.

Could use similar wording here?